structure/extension/_TAG_ref.yaml

```diff
@@ -0,0 +1,48 @@
+%YAML 1.2
+---
+lang: en-US
+
+type: structure
+
+uri: https://github.com/glamberson/gedcom-tags/_TAG_ref
+
+extension tags:
+  - _TAG
+
+specification:
+  - Tag Reference
+  - |
+    When used as a substructure, _TAG references a tag record to apply
+    that tag to the containing record. The payload is a pointer to a
+    _TAG record. Optional DATE and NOTE substructures can provide
+    context about when and why the tag was applied.
+
+label: 'Tag Reference'
+
+payload: "@<https://github.com/glamberson/gedcom-tags/_TAG>@"
+
+substructures:
+  "https://gedcom.io/terms/v7/DATE": "{0:1}"
+  "https://gedcom.io/terms/v7/NOTE": "{0:M}"
+
+superstructures:
+  "https://gedcom.io/terms/v7/record-INDI": "{0:M}"
+  "https://gedcom.io/terms/v7/record-FAM": "{0:M}"
+  "https://gedcom.io/terms/v7/record-SOUR": "{0:M}"
+  "https://gedcom.io/terms/v7/record-REPO": "{0:M}"
+  "https://gedcom.io/terms/v7/record-OBJE": "{0:M}"
+  "https://gedcom.io/terms/v7/record-NOTE": "{0:M}"
+  "https://gedcom.io/terms/v7/record-SNOTE": "{0:M}"
+  "https://gedcom.io/terms/v7/record-SUBM": "{0:M}"
+  "https://gedcom.io/terms/v7/EVEN": "{0:M}"
+  "https://gedcom.io/terms/v7/BIRT": "{0:M}"
+  "https://gedcom.io/terms/v7/DEAT": "{0:M}"
+  "https://gedcom.io/terms/v7/MARR": "{0:M}"
+  "https://gedcom.io/terms/v7/CENS": "{0:M}"
+  "https://gedcom.io/terms/v7/PLAC": "{0:M}"
+  "https://gedcom.io/terms/v7/NAME": "{0:M}"
+  "https://gedcom.io/terms/v7/ASSO": "{0:M}"
+
+contact: "lamberson@yahoo.com"
+
```

Comment:
if this is in use by gramps as you said:
> Actively tested with GRAMPS implementation

Then add `used by` as well to these YAML files.